File: A_mean.c
#include<stdio.h>
int main(){
    int N=0, i=0;
    float somma = 0, M=0;
    float dati[100];

    printf("\nInserisci valori: ");
    scanf("%d", &N);
    
    printf("\nInserisci i valori:");
    for (i = 0; i < N; i++)
    {
        scanf("%f", &dati[i]);
    }

    for (i = 0; i < N; i++)
    {
        somma += dati[i];
    }

    M = somma/N;
    printf("\nLa media e': %f", M);    
    
}

File: A_mean.py
from array import *

i = N = int(0)
num = somma = media = float(0)

dati = array('f')

N = int(input("Quanti numeri vuoi inserire?: "))
print("Inseriscili:")

for i in range(N):
    num = float(input(""))
    dati.append(num)

somma = sum(dati)
media = somma/N

print("La media dei numeri è: ", media)